contracts/fork-test.sh

```diff
@@ -77,7 +77,11 @@ main()
 
     # Run specific files when a param is given
     if [[ ! -z "$1" ]]; then
-        params+="--testfiles $@"
+        if [[ $is_coverage == "true" ]]; then
```

Comment:
Were local fork tests not working?

Reply:
Yes, `--testfiles` is only supported by `solcoverage` (the hardhat plugin we use for coverage tests). non-coverage tests don't support that param